core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-properties/dot-container-properties.component.ts

```diff
@@ -54,8 +54,7 @@ export class DotContainerPropertiesComponent implements OnInit, AfterViewInit {
                     title: new FormControl(container?.title ?? '', [Validators.required]),
                     friendlyName: new FormControl(container?.friendlyName ?? ''),
                     maxContentlets: new FormControl(container?.maxContentlets ?? 0, {
-                        validators: [Validators.required],
-                        updateOn: 'blur'
+                        validators: [Validators.required]
```

Comment:
is there a way to test this?

Reply:
Is this going to be triggered on every change of the field? is so... we need to catch that somehow.

Reply:
Yes, Noted

Reply:
@zulqarnainvd Isn't this updating the UI several time after it goes from 1 to 2 and 3 etc? I mean... we should only change tha state when 0 or 1. Can you double check that?